pages/notifications/manage-notifications/index.tsx

```diff
@@ -69,10 +71,21 @@ async function mergeGrantNameIntoSubscriptions(subscriptions) {
 
     return subscription;
   });
-}
+};
+
+const checkOneLoginEnabled = () => process.env.ONE_LOGIN_ENABLED === 'true';
```

Comment:
Really minor point but this doesn't really need to be a function

```suggestion
const ONE_LOGIN_ENABLED = process.env.ONE_LOGIN_ENABLED === 'true';
```

Reply:
yeah, I did it this way so it could be tested on and off without having to re-import `getServerSideProps`